proto/buf/alpha/module/v1alpha1/module.proto

```diff
@@ -34,6 +34,10 @@ message Module {
   //
   // string is used to enforce UTF-8 encoding or 7-bit ASCII text.
   string documentation = 3;
+  // breaking_config is the breaking change detection configuration set for the module.
+  BreakingConfigV1 breaking_config = 5;
+  // lint_config is the lint configuration set for the module.
+  LintConfigV1 lint_config = 6;
```

Comment:
I'll need to spend some more time on exactly how we should do this, but we might want to lean on a `package` name to distinguish difference versions from each other. So we'd have multiple messages called `BreakingConfig`, but they'd be placed in different packages with different versions, if ever. For example, `buf.module.v1.BreakingConfig` and `buf.module.v2.BreakingConfig`, etc.

Like I said, we probably need to think through this a bit more, but I'm not actually sure we want to include the version suffix in the message name itself.

Reply:
If we split these out, we could model it as `buf.lint.v1.Config` and `buf.breaking.v1.Config` defined in `proto/buf/{lint,breaking}/v1/config.proto`, respectively. We've already cut `v1` for these configurations in the `buf.yaml`, so we should feel free to reserve the same version for the Protobuf-equivalent.

> We _might_ still want to include these definitions in the `alpha` sub-package path, but based on the `buf.yaml` `v1` release point above, we might not need to.

We could reference those types from the `Module` definition, and support a `oneof` later if we ever need to make a `v2` for either of the configurations (i.e. a `oneof` that contains both `buf.lint.{v1,v2}.Config`).

This would mean that the package that defines the `Module` would be independent from the package that defines the configuration it relies on. This could be useful if we ever have another system that needs to consume the `lint` and `breaking` configuration without the `Module` definition, too.